documentation/modules/quickstart/proc-downloading-product.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// assembly-evaluation.adoc
+
+[id='proc-product-downloads-{context}']
+= Downloading {ProductName}
+
+A ZIP file contains the resources required for installation of {ProductName}, along with examples for configuration.
```

Comment:
"A ZIP file contains..." sounds a little generic to me, and I'm not sure about . You could reword to:

"The resources and artifacts required to install {ProductName}, along with examples for configuration, are provided in a ZIP file."

Reply:
Better :thumbsup: